src/app/containers/ChartbeatAnalytics/utils/index.js

```diff
@@ -146,7 +146,7 @@ export const getConfig = ({
     sections,
     uid: chartbeatUID,
     title,
-    virtualReferrer: decodeURIComponent(referrer),
+    virtualReferrer: referrer && decodeURIComponent(referrer),
```

Comment:
🤔 what happens when the `virtualReferrer` is `undefined`? do we have error handling for that?

Reply:
I would expect the `virtualReferrer` prop to just not be passed when undefined, and I believe this is expected behavior as this is what is happening, before the changes on this PR: https://github.com/bbc/simorgh/pull/6254/files#diff-bcb3577f4c9c42fa626f7f23b9c4f1c4R139

Reply:
I think it might be cleaner to not introduce an `undefined` state. Setting the `virtualReferrer` to null when it isn't available will be consistent with the previous design before the bug was introduced and also consistent with the output of the `getReferrer` method